less/pages/encrypt/hybrid-test.less

```diff
@@ -0,0 +1,7 @@
+.encrypt.v3.remove-fields.video4 .page .videoSection .encrypt-signup .signupForm {
```

Comment:
Nope, I'm just being safe because I don't really know what other classnames exist and how much is shared.

I suspect you're the expert on this, and if I remove the explicit path is it going to modify other pages in an unexpected way?